cli/tests/test_rebase_command.rs

```diff
@@ -589,10 +592,11 @@ fn test_rebase_multiple_revisions() {
     │ │ ◉  g
     ╭─┬─╯
     │ ◉  e
-    ◉ │  c
     │ │ ◉  h
     │ │ ◉  f
+    ╭───┤
     │ │ ◉  d
+    ◉ │ │  c
```

Comment:
Thanks - these new behaviors look reasonable and are in line with my mental model of how this command should behave. (I haven't looked at the `test_rebase_revisions_after` ones yet - I'll get back once I've wrapped my mind around how the "before" and "after" features should behave.)

Reply:
To close the loop on this, the `test_rebase_revisions_after` changes look reasonable. Thanks for including new test cases that test what happens when a target commit's parent turns out to be a descendant of a new child - having a specific rule for this case seems a bit inelegant, but I can't think of anything better.